site/source/developer-docs/getting-started/environment-setup.rst

```diff
@@ -73,7 +73,7 @@ To install the SDK:
 
 .. code:: bash
 
-    git clone https://github.com/Start9Labs/embassy-os.git
+    git clone --recursive https://github.com/Start9Labs/embassy-os.git && cd embassy-os/backend && ./install-sdk.sh
```

Comment:
whoops I messed this up actually, the last two lines need to be removed now,  but I can't edit your fork. @spencerflagg please do update, thanks!

Reply:
:+1: